addons/l10n_zm/data/template/account.account-zm.csv

```diff
@@ -0,0 +1,87 @@
+"id","name","code","account_type","reconcile"
+"zm_account_1000000","Sales","1000000","income","False"
+"zm_account_2000000","Cost of Sales / Purchases","2000000","expense_direct_cost","False"
+"zm_account_2100000","Inventory Adjustment","2100000","expense_direct_cost","False"
+"zm_account_2150000","Inventory Count Variance","2150000","expense_direct_cost","False"
+"zm_account_2200000","Purchase Variance","2200000","expense_direct_cost","False"
+"zm_account_2400000","Recovery Account","2400000","expense_direct_cost","False"
+"zm_account_2400010","Labour Cost Manufacturing Variance","2400010","expense_direct_cost","False"
```

Comment:
24000000 is really different than 2400010 and not just meant as account group?

Reply:
Not sure what i understand what you mean. You feel like 24...1, 24...2 and 24...3 should be out of 24, like in 25 for example ?

Reply:
- If you change the code to 240001 it will have the same effect as 2400010
- Checking in the spec, 20 is probably not a parent of 21, so it seems fine we have account 2000000 next to 2100000

Reply:
So no changes needed from the original 2400000, 2400010, 2400020, 2400030 but with less zero so 24, 240001, 240002 and 240003 ?

Reply:
We can do that.  Maybe 240000 for clarity still.